data/json/itemgroups/SUS/evac_shelter.json

```diff
@@ -3,16 +3,17 @@
     "id": "SUS_evac_shelter_locker",
     "type": "item_group",
     "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
-    "//2": "Each locker contains by spec one jacket, one blanket, one gas mask, a flashlight, a plastic bowl and utensils, a lighter, and two bottles of water.  Not every locker is fully stocked.",
+    "//2": "Each locker contains by spec one jacket, one blanket, one gas mask, a flashlight, a whistle, a plastic bowl and utensils, a lighter, and two bottles of water.  Not every locker is fully stocked.",
     "subtype": "collection",
     "entries": [
-      { "item": "evac_pamphlet", "prob": 99 },
-      { "item": "jacket_evac", "prob": 75 },
-      { "item": "emer_blanket", "prob": 75 },
+      { "item": "evac_pamphlet", "prob": 95 },
+      { "item": "jacket_evac", "prob": 72 },
+      { "item": "emer_blanket", "prob": 72 },
       { "item": "mask_gas", "prob": 2 },
-      { "item": "flashlight", "prob": 50 },
+      { "item": "flashlight", "prob": 48 },
```

Comment:
I'm not clear on why you're tweaking the probabilities here by such tiny percentages. This is another case of "if it's not really a change, don't change it just for the sake of it"

Reply:
I don't know that the math actually works like this, but I was worried about "collection" probability. I wanted to allow for new items to spawn, without decreasing the likelihood of existing items spawning (ie have my cake and eat it too).

To illustrate my concern let's say, for example, I add some bit of trash "advent burger wrapper" and make it ubiquitous. It then (to some degree) reduces the amount of items spawning that are more central to the intentional game mechanics.

In this instance I think I added a whistle with prob 12, and then (arbitrarily of course) removed 12 "points" of prob from other items.  Not because I had actually done the math - too interested in content and other concerns to bother with this degree of precision - but because I hoped my tweaks would be close enough. And the "Test Item Group" debug tool seemed to confirm that they were.

None of this is to dismiss your question. It is a fine question, and my reply is merely to give context on the problem without resolving it. Clearly these tweaks can be jettisoned if they aren't useful.

Reply:
Collection spawning probabilities don’t work like that. 

The game goes through the whole list. With every entry having an independent chance/100  probability of being spawned.

Removing the 12 points from the other entries is unnecessary.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/ITEM_SPAWN.md#collection-or-distribution